dom0/sd-journalist.sls

```diff
@@ -36,8 +36,12 @@ require:
     - group: root
     - mode: 644
 
-# Temporary workaround to bootstrap Salt support on target.
-qvm-run -a whonix-ws-14 "sudo apt-get install -qq python-futures":
+# Temporary workarounds for sd-journalist:
+#
+#   * python-futures required bootstrap Salt support
+#   * python-qt4 required for sd-process-feedback GUI integration
+#
+qvm-run -a whonix-ws-14 "sudo apt-get install -qq python-futures python-qt4":
```

Comment:
This is a nit .. but can we use the salt paradigm of 

```yaml
install qt4:
  cmd.run:
    - name: qvm-run -a whonix-ws-14 "sudo apt-get install -qq python-futures python-qt4"
```

instead?

Reply:
word, testing this PR with this suggested change